README.md

```diff
@@ -1,3 +1,18 @@
 # spfio-songdatabase
-# This app will help you prepare worship during sunday service. No more copy and paste. Just click and it will get the slides ready for you.
-# Website link: https://glorify.cc/
+## This app will help you prepare worship during sunday service. No more copy and paste. Just click and it will get the slides ready for you.
+## Website link: https://glorify.cc/
+
+# dev setup
```

Comment:
I think if you want to provide the guide for dev setup, you need to explain more like: "installing node" -> "installing gulp" -> "run command 'gulp' and you can open the web app in localhost:8000", etc